cmd/cstor-volume-grpc/api/api.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package api;
+message VolumeCommand {
```

Comment:
Comments about the fields would make it more readable.

Reply:
Can we have this code at `pkg/apis/openebs.io/v1alpha1/cstor-volume.proto` namespace?
I am not too sure of the file name I have proposed. However, do spend some thoughts here.
I have also assumed a version i.e. v1alpha1; please think about it.

Reply:
moved the .proto file to v1alpha1 package and renamed to cstorvolume.proto

Reply:
will send another pr with all the changes into snapshot branch.